src/assets/js/index.js

```diff
@@ -122,13 +122,6 @@ const defaultPolyfillBundleOptions = {
 
 const polyfillBundleOptions = Object.assign({}, defaultPolyfillBundleOptions);
 
-const featuresCache = {};
-const featuresRows = [...document.querySelectorAll("#features-list [data-feature-name]")];
-
-featuresRows.forEach(node => {
-	featuresCache[node.dataset.featureName.toLowerCase()] = node;
-});
```

Comment:
This was keeping references to the original list, breaking the filter after changing the version